inventory/host_vars/unifi.freifunk-mwu.de

```diff
@@ -0,0 +1,4 @@
+---
+server_type: "service"
+
+magic: 260
```

Comment:
Die `magic` muss noch auf einen Wert < 255 geändert werden. Momentan erzeugen einige Rollen unbrauchbare Ausgaben. z.B. spuckt die Rolle für das Loopback Interface `False` als IPv4 Adresse aus.

Reply:
Hm, würde ich ja gerne machen. Aber eigentlich ist das 10.X.0.0/24 ja für gateways reserviert. Oder ignorieren wir das mal?

Reply:
Ok.

Reply:
Die neuen Netzbereiche gibts hier: https://md.freifunk-mwu.de/domain-split#Netzbereiche1
Das Netz 10.87.255.0/24 ist für Loopbacks vorgesehen. Im Normalfall hat ein Dienste-Server nur eine Loopback in diesem Netz, auf die man aus den Domain geroutet zugreifen kann. Wenn ein Dienst ein Layer2-Bein in einer oder mehreren Domains benötigt, sieht das natürlich anders aus.